screenpipe-app-tauri/src-tauri/src/commands.rs

```diff
@@ -213,3 +213,23 @@ pub fn update_show_screenpipe_shortcut(
 
     Ok(())
 }
+
+#[tauri::command]
+pub fn open_accessibility_preferences() {
+    #[cfg(target_os = "macos")]
+    std::process::Command::new("open")
+        .arg("x-apple.systempreferences:com.apple.preference.security?Privacy_Accessibility")
+        .spawn()
+        .expect("failed to open system preferences");
+}
+
+#[tauri::command]
+pub fn reset_accessibility_permissions() {
+    #[cfg(target_os = "macos")]
+    std::process::Command::new("tccutil")
+        .arg("reset")
+        .arg("Accessibility")
+        .arg("so.cap.desktop")
```

Comment:
this is wrong

use screenpipe correct address (dont know how you call this), this is Cap one